src/SourceBuild/tarball/content/src/source-mappings.jsonc

```diff
@@ -1,7 +1,18 @@
+// This file configures where the VMR synchronizes the sources from.
+// Each individual repository has its own mapping which says where the remote repo is,
+// what files are excluded from the sync, etc.
+// More details can be found at: {{ TODO }}
 {
+    // Location withing the VMR where the source-build patches are stored
+    // These patches are applied on top of the code synchronized into the VMR
     "patchesPath": "src/installer/src/SourceBuild/tarball/patches",
+
+    // These defaults are added to all mappings unless `ignoreDefaults: true` is specified
+    // When no "include" filter is specified, "**/*" is used
```

Comment:
Would it make sense to have this directly specified in the `defaults` section instead of this comment?

Reply:
Unfortunately, that would mean something else.

The defaults are appended onto whatever filters the repo adds. If the repo doesn't set any, we use "all files" like this doc says. If we added it here, repos couldn't restrict anything as the "all files" filter would be added always.

I realize this is not intuitive and will be mentioned in the documentation later.